channels/bo.m3u

```diff
@@ -9,29 +9,13 @@ https://mediacp.hostradios.com.ar:19360/atbcochabamba/atbcochabamba.m3u8
 http://boliviatv1.srfms.com:5735/live/livestream/playlist.m3u8
 #EXTINF:-1 tvg-id="BoliviaTV72.bo" tvg-name="Bolivia TV 7.2" tvg-country="BO" tvg-language="Spanish" tvg-logo="https://i.imgur.com/1zTLGdj.jpg" group-title="",Bolivia TV 7.2 (720p)
 https://video1.getstreamhosting.com:1936/8224/8224/chunklist_w1684483949.m3u8
-#EXTINF:-1 tvg-id="BoliviaTV72.bo" tvg-name="Bolivia TV 7.2" tvg-country="BO" tvg-language="Spanish" tvg-logo="https://i.imgur.com/1zTLGdj.jpg" group-title="",Bolivia TV 7.2 (720p)
-https://video1.getstreamhosting.com:1936/8224/8224/playlist.m3u8
```

Comment:
works

Reply:
It's because the playlist only has got the chunklist above, not useful to keep it.